src/Vixen.Application/SetupDisplay/ViewModels/PropNodeTreeViewModel.cs

```diff
@@ -37,17 +48,22 @@ public PropNodeTreeViewModel()
 			PropComponentNodeViewModels = [pcvm];
 
 			PropNodes = new();
-			PropNodes.AddRange(RootNodeViewModel.SelectMany(x => x.GetPropEnumerator()));
+			Catel.Collections.CollectionExtensions.AddRange(PropNodes, RootNodeViewModel.SelectMany(x => x.GetPropEnumerator()));

```

Comment:
While this may compile, extension methods are meant to be called like member function.  I don't think this line of code should have been changed.